packages/app/src/components/DBTimeChart.tsx

```diff
@@ -149,10 +149,21 @@ export function DBTimeChart({
     const to = add(clickedActiveLabelDate, {
       seconds: convertGranularityToSeconds(granularity),
     }).getTime();
+    let where = config.where;
+    let whereLanguage = config.where || 'lucene';
+    if (where.length === 0 && Array.isArray(config.select)) {
```

Comment:
should we extract where clause from aggCondition only when there is a single series?